website/docs/r/cluster.html.markdown

```diff
@@ -424,7 +424,8 @@ replication_specs {
 
 **Replication Spec**  - Configuration block for multi-region cluster.
 
-* `num_shards` - (Required) Number of shards to deploy in the specified zone, minimum 1.
+* `num_shards` - (Required) Number of shards up to 50 to deploy for a sharded cluster. The resource returns 1 to indicate a replica set and values of 2 and higher to indicate a sharded cluster. The returned value equals the number of shards in the cluster.
```

Comment:
updating the description of the field to align with [API docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v1/#tag/Clusters/operation/createLegacyCluster)